### ISSUE TITLE
Instructions for adding a Google Calendar fall apart and do not apply

### ISSUE THREAD

**Issue by RickKukiela** [kewisch/gdata-provider#259]
### Describe the problem and steps to reproduce it:
Im using Thunderbird 94.0b1 (64-bit)

When I try to follow the instructions to add my google calendar after adding the extension It says to click new calenedar, then "on the network", then select "Google Calendar"... However after clicking on the network there is no option for choosing google calendar:

![image](https://user-images.githubusercontent.com/3813238/136429220-1c592985-78e3-4747-af40-8c188fc7d33a.png)

Halp!

(Please include as many details as possible.)

**Comment by kewisch** [comment]
When you open the new calendar wizard, there should be three options: On my computer, on the network, and Google Calendar. If you select On the Network it won't take you to the right spot. Is there no direct option for Google Calendar?

Do you get any error console messages?

**Comment by RickKukiela** [comment]
Wow, so not having that option made me go and double check the extension and somehow it wasn't even installed. Go figure. Not sure how that was the case since I know I installed it... Anyway, I installed it again and now it is working as described. I did have to restart thunderbird after installing it to get the option for google to appear.... also after I entered my email address to start a new session it hung while it was supposed to open the o-auth google sign in process... I found out thats what it was supposed to do after I cancelled it and tried it again and then it seemed to work as it was supposed to. You might want to put some time out on that and do a little more to inform the user what is happening and what they should expect to happen next in case it doesnt work... Obviously addressing whatever makes it unreliable in the first place would also be recommended just from a "User feedback" perspective. In any event it seems to be working perfectly now... If only I can figure out how to make thunderbird use one of my google calendars when I accept event event invites emailed to me haha... off to google!
